src/main/resources/harmony.mixins.json

```diff
@@ -24,5 +24,5 @@
   ],
   "injectors": {
     "defaultRequire": 1
-	}
```

Comment:
Crucial change definetly.
LGTM

Reply:
somehow both harmony and meldoy had this issue?